packages/venia-concept/venia-upward.yml

```diff
@@ -46,18 +46,14 @@ imageOpt:
     headers:
       inline:
         Location:
-          engine: mustache
-          provide:
-            width: $match.$1
-            path: request.url.query.url
-          template:
-            when:
-              - matches: env.USE_FASTLY
-                pattern: '[^0]'
-                use:
-                  inline: '{{path}}?auto=webp&format=pjpg&width={{width}}'
-            default:
-              inline: '{{path}}'
+          baseUrl: request.url.query.url
```

Comment:
You're comfortable hard-coding Fastly params into all upward-php  image requests (or upward-js w/ imageopt middleware disabled)?

Reply:
I'm not super comfortable with it, but that's what we can do today, lacking proper URL rewrite. We could maybe extend the ProxyResolver to be a rewrite engine with certain arguments.

Reply:
Fahd has requested the ability to inject headers into Proxy requests as well, so let's leave this as-is, and include rewrite functionality in a larger Proxy improvement story. Let's just make sure @dpatil-magento is aware of this change so he can check these query params won't cause any unexpected behavior when Fastly isn't being used (seems unlikely).